components/Vote/Vote.tsx

```diff
@@ -1,31 +1,41 @@
-import { useCallback, useState } from "react";
+import { Dispatch, SetStateAction, useCallback } from "react";
 import cx from "classnames";
-
+import React from "react";
+type voteProps = {
+  applicationId: Number;
+  voteCount: number;
+  isUserAuthenticated: Boolean;
+  variant: String;
+  setVoteCount: Dispatch<SetStateAction<number>>;
+};
 // TODO: When transitioning to typescript, variant can be either 'vibrant' or 'simple'
-export default function Vote({ applicationId, voteCount, isUserAuthenticated, variant, setVoteCount }) {
-  const [isLoading, setLoading] = useState(false);
-
+export default function Vote({ applicationId, voteCount, isUserAuthenticated, variant, setVoteCount }: voteProps) {
```

Comment:
What's the best practice here? I think I am defaulting to using the generic `React.FunctionComponent` type instead of plain functions.
```suggestion
const Vote: React.FunctionComponent<VoteProps> = ({ applicationId, voteCount, isUserAuthenticated, variant, setVoteCount) => {
```

Reply:
yes i thinks its a better way